src/renderer/components/Onboarding/Screens/Tutorial/screens/PinCodeHowTo.js

```diff
@@ -49,8 +49,8 @@ const StepIndexContainer = styled.div`
   display: flex;
   justify-content: center;
   align-items: center;
-  background: rgba(100, 144, 241, 0.1);
-  color: #6490f1;
+  background: rgba(138, 128, 219, 0.1);
+  color: #8a80db;
```

Comment:
What's up with all the color changes? 😕

Reply:
Not that I care but the scope of the task is supposed to be activating the page + porting some logic

Reply:
removes dusk mode and updates theme color `(to confirm with design if ok)`
we want to change the main color to match the new themes all other the app and not have discrepancy between market vs the rest.
the idea is ok with design expect the purple i put seems to be the wrong one :sweat_smile:

Reply:
So then it's wrong Luiz, if the color is wrong, it's all wrong  (╯°□°)╯︵ ┻━┻